chyt/README.md

```diff
@@ -0,0 +1,31 @@
+## 1. Requesting a Demo Cluster
+
+* Visit the [YTsaurus Getting Started webpage](https://ytsaurus.tech/docs/en/overview/try-yt#demo-stand) to book your demo cluster.
```

Comment:
When I try to create a demo cluster, the UI asks me for a date and time. What time zone is the time specified in?

![grafik](https://github.com/user-attachments/assets/24c4c66d-f43c-4854-b6b9-0c7f67bde9d5)

Reply:
Ah, I just see that the mail contains a time zone:

![grafik](https://github.com/user-attachments/assets/4f1757e2-9bf2-4864-ae6f-709ac8fedb64)